app/helpers/mobile_helper.rb

```diff
@@ -1,41 +1,40 @@
 module MobileHelper
-  def aspect_select_options(aspects, selected)
-    selected_id = selected == :all ? "" : selected.id
-    '<option value="" >All</option>\n'.html_safe + options_from_collection_for_select(aspects, "id", "name", selected_id)
-  end
-
   def mobile_reshare_icon(post)
     if (post.public? || reshare?(post)) && (user_signed_in? && post.author != current_user.person)
       absolute_root = reshare?(post) ? post.absolute_root : post
 
       if absolute_root && absolute_root.author != current_user.person
-        reshare = Reshare.where(:author_id => current_user.person_id,
-                                :root_guid => absolute_root.guid).first
+        reshare = Reshare.where(author_id: current_user.person_id,
+                                root_guid: absolute_root.guid).first
         klass = reshare.present? ? "active" : "inactive"
-        link_to '', reshares_path(:root_guid => absolute_root.guid), :title => t('reshares.reshare.reshare_confirmation', :author => absolute_root.author_name), :class => "image_link reshare_action #{klass}"
+        link_to "", reshares_path(root_guid: absolute_root.guid),
+                title: t("reshares.reshare.reshare_confirmation", author: absolute_root.author_name),
+                class: "entypo-reshare reshare-action #{klass}"
       end
     end
   end
 
   def mobile_like_icon(post)
     if current_user && current_user.liked?(post)
-      link_to '', post_like_path(post.id, current_user.like_for(post).id), :class => "image_link like_action active"
+      link_to "", post_like_path(post.id, current_user.like_for(post).id), class: "entypo-heart like-action active"
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.